src/EnergyPlus/Coils/CoilCoolingDXCurveFitSpeed.cc

```diff
@@ -184,15 +184,15 @@ void CoilCoolingDXCurveFitSpeed::instantiateFromInputSpec(EnergyPlus::EnergyPlus
             ShowContinueError(state, "..." + fieldName + "=\"" + curveName + "\" has out of range values.");
             ShowContinueError(state, format("...Curve minimum must be >= 0.7, curve min at PLR = {:.2T} is {:.3T}", MinCurvePLR, MinCurveVal));
             ShowContinueError(state, "...Setting curve minimum to 0.7 and simulation continues.");
-            Curve::SetCurveOutputMinMaxValues(state, this->indexPLRFPLF, errorsFound, 0.7, _);
```

Comment:
None of the calls to `SetCurveOutputMinMaxValues` actually used both the min and max returns, so I split this function into two and renamed appropriately.

Reply:
This seems fine.  I think this may change a bit more as I do my next deeper dive on the curve manager, looking especially at the "client" API, but this seems good for the intent of this Optional sweep.